services/core-web/common/components/documents/ArchivedDocumentsSection.tsx

```diff
@@ -16,6 +17,22 @@ const ArchivedDocumentsSection = (props: ArchivedDocumentsSectionProps) => {
     return <></>;
   }
 
+  const parseArchivedDocuments = () => {
+    return props.documents.map((obj) => ({
+      ...obj,
+      key: obj.mine_document_guid,
+      file_location_type_code: obj.major_mine_application_document_xref.major_mine_application_document_type_code
+        || obj.project_summary_document_xref.project_summary_document_type_code
```

Comment:
I'm thinking that when we set these attributes we can just set them as the file_location_type_code when we create the object and then not have to do a bunch of "OR"

Reply:
Updated.